src/rotest/common/django_utils/settings_common.py

```diff
@@ -1,47 +0,0 @@
-"""Common Django configuration constants.
```

Comment:
why did you delete all the settings? I think leaving one with the default behavior would be nicer for the average user who wants to avoid all the django crap